front/js/cart.js

```diff
@@ -1,92 +1,324 @@
+    productItemContentSettings.className = "cart__item__content__settings"
 
     // Insertion de l'élément "div"
-    let productItemContentSettingsQuantity = document.createElement("div");
-    productItemContentSettings.appendChild(productItemContentSettingsQuantity);
-    productItemContentSettingsQuantity.className = "cart__item__content__settings__quantity";
+    let productItemContentSettingsQuantity = document.createElement("div")
+    productItemContentSettings.appendChild(productItemContentSettingsQuantity)
+    productItemContentSettingsQuantity.className = "cart__item__content__settings__quantity"
     
     // Insertion de "Qté : "
-    let productQte = document.createElement("p");
-    productItemContentSettingsQuantity.appendChild(productQte);
-    productQte.innerHTML = "Qté : ";
+    let productQte = document.createElement("p")
+    productItemContentSettingsQuantity.appendChild(productQte)
+    productQte.innerHTML = "Qté : "
 
     // Insertion de la quantité
-    let productQuantity = document.createElement("input");
-    productItemContentSettingsQuantity.appendChild(productQuantity);
-    productQuantity.value = produitLocalStorage[product].quantiteProduit;
-    productQuantity.className = "itemQuantity";
-    productQuantity.setAttribute("type", "number");
-    productQuantity.setAttribute("min", "1");
-    productQuantity.setAttribute("max", "100");
-    productQuantity.setAttribute("name", "itemQuantity");
+    let productQtty = document.createElement("input")
+    productItemContentSettingsQuantity.appendChild(productQtty)
+    productQtty.value = productLocalStorage[product].productQuantity
+    productQtty.className = "itemQuantity"
+    productQtty.setAttribute("type", "number")
+    productQtty.setAttribute("min", "1")
+    productQtty.setAttribute("max", "100")
+    productQtty.setAttribute("name", "itemQuantity")
 
     // Insertion de l'élément "div"
-    let productItemContentSettingsDelete = document.createElement("div");
-    productItemContentSettings.appendChild(productItemContentSettingsDelete);
-    productItemContentSettingsDelete.className = "cart__item__content__settings__delete";
+    let productItemContentSettingsDelete = document.createElement("div")
+    productItemContentSettings.appendChild(productItemContentSettingsDelete)
+    productItemContentSettingsDelete.className = "cart__item__content__settings__delete"
 
     // Insertion de "p" supprimer
-    let productSupprimer = document.createElement("p");
-    productItemContentSettingsDelete.appendChild(productSupprimer);
-    productSupprimer.className = "deleteItem";
-    productSupprimer.innerHTML = "Supprimer";
+    let productSupprimer = document.createElement("p")
+    productItemContentSettingsDelete.appendChild(productSupprimer)
+    productSupprimer.className = "deleteItem"
+    productSupprimer.innerHTML = "Supprimer"
 }
 }}
-getCart();
\ No newline at end of file
+getCart()
+
+function getTotals(){
```

Comment:
i would advise you to get the total not from the html but from the localstorage! because you are first drawing the cart into html and then using html (with get elements by ids and so on) which make that if you change slightly your html this code will never work again. Opposed to that fact that if you decide to use localstorage you will rely on something that is less likely to change compared to html tags and page.